evalml/data_checks/null_data_check.py

```diff
@@ -180,6 +243,54 @@ def validate(self, X, y=None):
                 ).to_dict()
             )
 
+        if below_highly_null_cols:
+            results["warnings"].append(
+                DataCheckWarning(
+                    message="Columns {} have null values".format(
```

Comment:
Nit: would it make more sense to say "Column(s)" instead of "Columns"?

Reply:
Ah I was trying to make this consistent with the highly null message, but I'll just change both to use "Column(s)" instead 😂